io.github.ec_.Quake3e.OpenArena.desktop

```diff
@@ -3,7 +3,6 @@ Name=OpenArena (Quake3e)
 Comment=First person shooter
 Icon=io.github.ec_.Quake3e.OpenArena
 Exec=openarena-q3e
-Terminal=false
+Terminal=true
 Type=Application
 Categories=Game;ActionGame;
-PrefersNonDefaultGPU=true
```

Comment:
Also, why did you remove this? This is an important variable for all laptop users that have two GPUs. Without this, the io.github.ec_.Quake3e.OpenArena Flatpak will always run on an integrated, low-end GPU instead of the proper discrete one.

Reply:
Arent you supposed to set your default GPU to your dedicated one?

Reply:
Nope, default GPU is usually the integrated one since it has much lower power consumption. Dedicated one is used only for more demanding tasks, such as games.